### PR TITLE
Sync up

### PR DESCRIPTION
**Pull request by derekhiggins**
Syncing up patches from the downstream (openshift) image to here.

### CODE REVIEW

**Review by elfosardo — APPROVED** [review]

File: ironic.conf.j2 [hunk 1]
```diff
@@ -58,7 +58,7 @@ enable_ssl_api = true
 automated_clean = {{ env.IRONIC_AUTOMATED_CLEAN }}
 # NOTE(dtantsur): keep aligned with [pxe]boot_retry_timeout below.
 deploy_callback_timeout = 4800
-send_sensor_data = true
+send_sensor_data = false
```
Comment (dtantsur) [inline ironic.conf.j2:61]: I don't think we should change this.

File: main-packages-list.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+crudini
+dnsmasq
+gdisk
+genisoimage
+httpd
+httpd-tools
+ipmitool
+iproute
+iscsi-initiator-utils
```
Comment (dtantsur) [inline main-packages-list.txt:9]: I wonder why we have this, we don't use the iscsi deploy..

**Review by dtantsur — APPROVED** [review]

File: ironic.conf.j2 (hunk 1 above)
Comment (derekhiggins) [reply ironic.conf.j2:61]: should change it at all, or upstream?

**Pull request merged**